packages/drivers/routerlicious-driver/src/test/errorUtils.spec.ts

```diff
@@ -4,67 +4,78 @@
  */
 
 import assert from "assert";
-import { DriverErrorType, IThrottlingWarning } from "@fluidframework/driver-definitions";
+import { IThrottlingWarning } from "@fluidframework/driver-definitions";
+import { FluidErrorTypes } from "@fluidframework/core-interfaces";
 import {
 	createR11sNetworkError,
 	throwR11sNetworkError,
-	RouterliciousErrorType,
+	RouterliciousErrorTypes,
 	errorObjectFromSocketError,
 } from "../errorUtils";
 
 describe("ErrorUtils", () => {
+	/**
+	 * Checks if the input is an {@link IThrottlingWarning}.
+	 * */
+	function isIThrottlingWarning(input: unknown): input is IThrottlingWarning {
+		return (
+			(input as Partial<IThrottlingWarning>).errorType === FluidErrorTypes.throttlingError &&
+			(input as Partial<IThrottlingWarning>).retryAfterSeconds !== undefined
+		);
```

Comment:
For consistency with the others :)
```suggestion
		return (
			input !== undefined &&
			(input as Partial<IThrottlingWarning>).errorType === FluidErrorTypes.throttlingError &&
			(input as Partial<IThrottlingWarning>).retryAfterSeconds !== undefined
		);
```